common.blocks/page-index/page-index.css

```diff
@@ -0,0 +1,17 @@
+.page-index {
+    display: flex;
+    flex-direction: row;
+
+    &__sidebar {
+        padding: 20px;
+        min-width: 180px;
+        background-color: #f3f1ed;
+        border-left: 1px solid rgba(0, 0, 0, .1);
+    }
+
+    &__content {
+        flex-grow: 1;
+        padding-bottom: 20px;
+        overflow: auto;
```

Comment:
the problem is not only with images, but with long code too
so i think it would be better to leave overflow: auto for now
and later, when we will do post design - we will fix it